frontend/simple/js/state.js

```diff
@@ -130,14 +156,15 @@ const actions = {
       return console.error(`UNKNOWN EVENT TYPE!`, contractId, entry)
     }
     // handleEvent might be called very frequently, so save only after a pause
-    debouncedSave(dispatch)
+    debouncedSave(dispatch, state.loggedIn)
```

Comment:
Wait, if you're just always passing in something that exists in the state, then there's no reason to pass it in as an argument since the function already has the state being passed into it.